### PR TITLE
change readme to emphasize SSO

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
-- Implementing a WebAPI to [retrieve attachments via a callback token and the Outlook REST API](https://dev.office.com/docs/add-ins/outlook/use-rest-api)
 - [Using the SSO access token](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) to call the Microsoft Graph API without prompting the user
 - If the SSO token is not available, authenticating to the user's OneDrive using the OAuth2 implicit flow via the [office-js-helpers library](https://github.com/OfficeDev/office-js-helpers).
 - Using the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/api-reference/v1.0/resources/onedrive) to create files in OneDrive.
+- Adding [add-in command buttons](https://docs.microsoft.com/outlook/add-ins/add-in-commands-for-outlook) to the Outlook ribbon when reading mail, including a UI-less button and a button that opens a task pane
+- Implementing a WebAPI to [retrieve attachments via a callback token and the Outlook REST API](https://dev.office.com/docs/add-ins/outlook/use-rest-api)
```
Comment (ElizabethSamuel-MSFT) [inline ReadMe.md:20]: @Rick-Kirkham Where does this link resolve to (since dev.office.com has been deprecated)?

Comment (Rick-Kirkham) [reply ReadMe.md:20]: It should now go to https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/implement-sso-in-outlook-add-in?branch=outlook-sso-sample. I'll fix it.

Comment (ElizabethSamuel-MSFT) [reply ReadMe.md:20]: You should update it to https://docs.microsoft.com/office/dev/add-ins/outlook/implement-sso-in-outlook-add-in. Thx.

Comment (Rick-Kirkham) [reply ReadMe.md:20]: Actually, now that I read the intro phrase carefully, it should go to get-attachments-of-an-outlook-item.md

**Pull request merged**